src/clj/tic_tac_toe_clojure/core.clj

```diff
@@ -12,22 +12,33 @@
 (defn create-board []
   (into [] (repeat 9 "_")))
 
+(defn create-4x4-board []
+  (into [] (repeat 16 "_")))
+
 (def numbered-board
   ["0" "1" "2" "3" "4" "5" "6" "7" "8"])
 
+(def numbered-4x4-board
+  ["0" "1" "2" "3" "4" "5" "6" "7" "8" "9" "10" "11" "12" "13" "14" "15"])
```

Comment:
Any chance to delete some duplication? 

Let say, create a method that returns an array with their id?
```
(def create-board-with-number [size]
   ...)
```